core/tx_pool.go

```diff
@@ -1026,6 +1075,7 @@ func (pool *TxPool) removeTx(hash common.Hash, outofbound bool) {
 	// Remove the transaction from the pending lists and reset the account nonce
 	if pending := pool.pending[addr]; pending != nil {
 		if removed, invalids := pending.Remove(tx); removed {
+			pool.pendingCount--
```

Comment:
Is it thread safe variable?

Reply:
it should be, as far as it has the same level of safeness as pool.pending.